File: Application programming Java/generic/SpecificOOP.java
import java.lang.*;

class Arithmetic
{
	 public int No1,No2;
	
	 public Arithmetic(int a, int b)
	 {
		 No1 = a;
		 No2 = b;
	 }
	 public int Addition()
	 {
		 int Ans = 0;
		 Ans = No1 + No2;
		 return Ans;
	 }
}
class SpecificOOP
{
	public static void main(String arg[])
	{
		int ret = 0;
		Arithmetic obj = new Arithmetic(10,11);
		
		ret = obj.Addition();
		
		System.out.println("Addition is :"+ret);
	}
}

File: Application programming Java/generic/SpecificOOP.cpp
#include<iostream>
using namespace std;


class Arithmetic
{
	public:
	 int No1;
	 int No2;
	 
	 Arithmetic( int a, int b)
	 {
		 No1 = a;
		 No2 = b;
	 }
	 
	 int Addition()
	 {
		 int ans;
		 ans = No1 + No2;
		 return ans;
	 }
	 
	 int Substraction()
	 {
		 int ans;
		 ans = No1 - No2;
		 return ans;
	 }	 
};

int main()
{
	  
	 Arithmetic iobj(10,20);    
	 
	 int ret = iobj.Addition();
	 cout<<"The integer nos addition is :"<<ret<<"\n";
	  
	 return 0;
}
